src/webrtc/stats/trackStatsReporter.ts

```diff
@@ -153,4 +153,17 @@ export class TrackStatsReporter {
             });
         return { audioTrackSummary, videoTrackSummary };
     }
+
+    public static buildJitter(trackStats: MediaTrackStats, statsReport: any): void {
+        if (statsReport.type !== "inbount-rtp") {
```

Comment:
```suggestion
        if (statsReport.type !== "inbound-rtp") {
```

I think this is a typo